chapters/equations.tex

```diff
@@ -475,15 +475,15 @@ \subsection{assert}\label{assert}
 assert(condition, message, assertionLevel);
 assert(condition, message, level = assertionLevel);
 \end{lstlisting}
-Here, \lstinline!condition! is a \lstinline!Boolean! expression, \lstinline!message! is a \lstinline!String! expression, and \lstinline!assertionLevel! is an optional parameter expression of the built-in enumeration type \lstinline!AssertionLevel!.
+Here, \lstinline!condition! is a \lstinline!Boolean! expression, \lstinline!message! is a \lstinline!String! expression, and \lstinline!assertionLevel! is an optional evaluable expression of the built-in enumeration type \lstinline!AssertionLevel!.
 It can be used in equation sections or algorithm sections.
 
 \begin{nonnormative}
 This means that \lstinline!assert! can be called as if it were a function with three formal parameters, the third formal parameter has the name \lstinline!level! and the default value \lstinline!AssertionLevel.error!.
 \end{nonnormative}
 
 \begin{nonnormative}
-A parameter expression is required for \lstinline!level! since it shall be evaluated at compile time.
+An evaluable expression is required for \lstinline!level! since it shall be evaluated at compile time.
```

Comment:
This goes without saying now.  I suggest removing the entire non-normative section.

Reply:
Removed.